nuitka/plugins/standard/PlaywrightPlugin.py

```diff
@@ -0,0 +1,168 @@
+#     Copyright 2024, Kay Hayen, mailto:kay.hayen@gmail.com find license text at end of file
+
+""" Plugin for Playwright.
+
+spell-checker: ignore Playwright
+"""
+
+from nuitka.Options import isStandaloneMode
+from nuitka.plugins.PluginBase import NuitkaPluginBase
+import os
+from nuitka.utils.Execution import NuitkaCalledProcessError
+
+
+class NuitkaPluginPlaywright(NuitkaPluginBase):
+    """This class represents the main logic of the plugin."""
+
+    plugin_name = "Playwright"
+    plugin_desc = "Required by 'playwright' package."
+
+    def __init__(self, include_browsers):
+        self.include_browsers = list(include_browsers)
+        self.installed_browsers = self.get_Installed_Playwright_Browsers()
```

Comment:
Always enabled plugins cannot run code during init. I think you have to treat empty "include_browsers" special and NOT check for installed browsers. Pretty sure that should potentially crash when playwright is not installed.